src/components/ListItem/ListItem.vue

```diff
@@ -400,6 +402,12 @@ export default {
 		handleActionsUpdateOpen(e) {
 			this.menuOpen = e
 		},
+
+		// Check if subtitle and actions slots is populated
+		checkSlots() {
+			this.hasActions = !!this.$slots.actions
+			this.hasSubtitle = !!this.$slots.subtitle
```

Comment:
Only if they're different from `this.hasActions` or `this.hasSubtitle`.
or else, endless loop! :warning:

Reply:
done!